CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [Unreleased]
+
+### Fixed
+* Fixed a bug where `videoTileSizeDidChange` is called even when frame is null
```

Comment:
Same comment for the iOS change, should this be a breaking change?

Also, probably rephrase this as something like "Fixed a bug that `videoTileSizeDidChange()` being called with width=0 and size=0 when the video was paused", as builders may not have insights on what does "frame is null" means. Same for Android changelog.

Reply:
updated